internal/aws/elbv2.go

```diff
@@ -322,3 +323,23 @@ func (c *Cloud) describeTargetGroupsHelper(input *elbv2.DescribeTargetGroupsInpu
 	})
 	return result, err
 }
+
+// PopulateTargetAZ populates the AvailabilityZone field of the provided TargetDescriptions
+func PopulateTargetAZ(ctx context.Context, c CloudAPI, a []*elbv2.TargetDescription) error {
+	vpc, err := c.GetVpcWithContext(ctx)
+	if err != nil {
+		return err
+	}
+
+	_, ipv4Net, err := net.ParseCIDR(*vpc.CidrBlock)
```

Comment:
we should use vpc.CidrBlockAssociationSet instead of vpc.CidrBlock. (some vpc have multiple CIDR blocks)